docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -198,11 +225,11 @@ State and Transition Data Tables
      - Source
      - Description
      - Notes
-   * - prevalence_c589
-     - como
-     - prevalence of chronic kidney disease
+   * - prevalence_c493
```

Comment:
I haven't double checked all of these---but I believe that for prevalence and incidence, the source is como! i was getting tables from get_outputs(cause_id=493). For deaths and emr, the source should be codcorrect---but i might be misunderstanding, if there's something wonky that goes on with them estimating IHD from the MI data?

Reply:
i'm seeing deaths from get_outputs:
![go_call](https://user-images.githubusercontent.com/57412727/73582789-91edcd80-4443-11ea-8fa2-b6fa69b1a1e4.png)

Reply:
Thank you for identifying that! Is there a standard checklist of databases to check, when decided the sources? I may have been looking at the IHD flowchart in Appendix 1 incorrectly!